src/Moryx.Model/Configuration/ModelConfiguratorBase.cs

```diff
@@ -139,14 +139,14 @@ public virtual async Task<DatabaseMigrationSummary> MigrateDatabase(IDatabaseCon
                 await context.Database.MigrateAsync();
                 result.Result = MigrationResult.Migrated;
                 result.ExecutedMigrations = pendingMigrations;
-                Logger.Log(LogLevel.Info, "Database migration for database '{0}' was successful. Executed migrations: {1}",
+                Logger.Log(LogLevel.Information, "Database migration for database '{0}' was successful. Executed migrations: {1}",
```

Comment:
Logger has LogInformation, LogWarning, etc.
I am not sure if changing log level will have the same effect

Reply:
`LogInformation` is just an extension that calls `Log(LogLevel.Information` underneath.

The change was from Moryx.LogLevel to MS.LogLevel, which are similar but not identical.